### PR TITLE
Restructure revisions section stories and enable snapshotting

### PR DESCRIPTION
**Pull request by fkling**
This addresses #30336, consolidates the stories for the revisions section and enables snapshots (is it important? 🤷🏻‍♂️ ).

Given the experimental nature of search stacks (and since it will change anyways soon enough), I don't think we need to enable snapshots there.

### CODE REVIEW

File: client/web/src/search/results/sidebar/Revisions.story.tsx [hunk 1]
```diff
@@ -24,107 +24,80 @@ import {
...
     component: Revisions,
-    decorators: [
-        Story => (
-            <div className={sidebarStyles.searchSidebar}>
-                <Story />
-            </div>
-        ),
-    ],
     argTypes: { onFilterClick: { action: 'onFilterClick' } },
+    disableSnapshot: false,
```
Comment (fkling) [inline client/web/src/search/results/sidebar/Revisions.story.tsx:28]: Is this where I have to put the flag?

Comment (limitedmage) [reply client/web/src/search/results/sidebar/Revisions.story.tsx:28]: @valerybugakov probably knows best, I think it has to be under `parameters.chromatic`.

Comment (valerybugakov) [reply client/web/src/search/results/sidebar/Revisions.story.tsx:28]: @fkling, here's [the documentation](https://www.chromatic.com/docs/ignoring-elements#ignore-stories) describing the flag. @limitedmage is correct; it should live under the `parameters.chromatic` property:

https://github.com/sourcegraph/sourcegraph/blob/4d0f8e1fcd7cd556a24a95b327000bf9835f789d/client/wildcard/src/components/Badge/ProductStatusBadge.story.tsx#L10-L28

**Review by valerybugakov — APPROVED** [review]
Thanks for looking into it! 🙌
Requires moving the `disableSnapshot` property to the appropriate place, and looks good otherwise.

**Pull request merged**